src/CommunityVoices/App/Website/Controller/Tag.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace CommunityVoices\App\Website\Controller;
+
+use CommunityVoices\App\Website\Component;
+
+class Tag
+{
+    //protected $tagAPIController;
+    protected $apiProvider;
+
+    public function __construct(
+        //Api\Controller\Tag $tagAPIController
+        Component\ApiProvider $apiProvider
+    ) {
+        //$this->tagAPIController = $tagAPIController;
+        $this->apiProvider = $apiProvider;
+    }
+
+    public function getAllTag($request)
+    {
+        //$this->tagAPIController->getAllTag($request);
+    }
+
+    public function getTagUpload($request)
+    {
+        //$this->tagAPIController->getTagUpload($request);
+    }
+
+    public function postTagUpload($request)
+    {
+        //$this->tagAPIController->postTagUpload($request);
+        $errors = $this->apiProvider->postJson('/tags/new', $request);
+        return $errors;
+    }
+
+    public function postTagDelete($request)
+    {
+        //$this->tagAPIController->postTagDelete($request);
+        $id = $request->attributes->get('groupId');
+        $errors = $this->apiProvider->postJson("/tags/{$id}/delete", $request);
+        return $errors;
```

Comment:
If you're going to return the errors, then you should probably display them rather than provide a dummy API response.

Reply:
I am a little confused what you mean here, isn't that what I am doing?

Reply:
Looks like this got added